dev/terria/terria-cube-v8.json

```
@@ -106,6 +106,90 @@
                             "type": "group",
                             "name": "DEA Surface Reflectance Calendar Year (Landsat)",
                             "members": [
+                                {
+                                    "type": "wms",
+                                    "name": "DEA GeoMAD (Landsat 8 & 9 OLI-TIRS) v4",
```

Assume we will drop this when we add the layers to prod DEA Maps? For now, nicer to put "v4" inside the brackets 

```suggestion
                                    "name": "DEA GeoMAD (Landsat 8 & 9 OLI-TIRS, version 4)",
```

---

@robbibt- yep version label will be dropped when the existing version is decommissioned - so only one will remain with no versioning in name